### PR TITLE
Fix edit booking bug

### PR DESCRIPTION
**Pull request by GERARDJM018**

### CODE REVIEW

**Review by LimZiJia — APPROVED** [review]
Nice catch! LGTM

File: src/main/java/seedu/address/logic/commands/EditHousekeepingDetailsCommand.java [hunk 1]
```diff
@@ -18,7 +14,16 @@
...
+import seedu.address.model.person.Address;
+import seedu.address.model.person.Area;
+import seedu.address.model.person.Booking;
+import seedu.address.model.person.Client;
+import seedu.address.model.person.Email;
+import seedu.address.model.person.HousekeepingDetails;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Phone;
+import seedu.address.model.person.Type;
```
Comment (gosongying) [inline src/main/java/seedu/address/logic/commands/EditHousekeepingDetailsCommand.java:26]: Nice.

File: src/main/java/seedu/address/logic/commands/EditHousekeepingDetailsCommand.java [hunk 2]
```diff
@@ -140,6 +147,9 @@ public String toString() {
     public static class EditHousekeepingDetailsDescriptor {
         private LocalDate lastHousekeepingDate;
         private Period preferredInterval;
+        private Period deferment;
+
```
Comment (gosongying) [inline src/main/java/seedu/address/logic/commands/EditHousekeepingDetailsCommand.java:151]: Perhaps can remove the space.

File: src/main/java/seedu/address/logic/parser/ParserUtil.java [hunk 3]
```diff
@@ -9,24 +9,22 @@
...
 import seedu.address.commons.util.StringUtil;
 import seedu.address.logic.parser.exceptions.ParseException;
-import seedu.address.model.person.Address;
-import seedu.address.model.person.Area;
-import seedu.address.model.person.Email;
-import seedu.address.model.person.HousekeepingDetails;
-import seedu.address.model.person.Name;
-import seedu.address.model.person.Phone;
-import seedu.address.model.person.Type;
+import seedu.address.model.person.*;
```
Comment (gosongying) [inline src/main/java/seedu/address/logic/parser/ParserUtil.java:17]: Better to import explicitly.

**Review by gosongying — APPROVED** [review]
LGTM.

**Pull request merged**